fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocksLex.g4

```diff
@@ -171,7 +174,7 @@ SINGLE_QUOTED_TEXT
     ;
 
 DOUBLE_QUOTED_TEXT
-    : '"' ( ~'"' | '""' )* '"'
+    : '"' (('\\' .)? ( ~'"' | '""' ))*? '"'
```

Comment:
Is this regex？

Reply:
This location is all antlr code, not regular expressions. But the semantics here are the same as regular expressions, which can be understood according to the semantics of regular expressions